doc-content/drools-docs/src/main/asciidoc/AuthoringAssets/decision-tables-defining-proc.adoc

```diff
@@ -30,7 +30,19 @@ NOTE: As an alternative to populating both the object type and constraint cells,
 If you need to add more rule tables to this decision table spreadsheet, skip a row after the last rule in the previous table, label another `RuleTable` cell in the same column as the previous `RuleTable` and  `RuleSet` cells, and create the new table following the same steps in this section (rows 19-29 in example).
 . Save your XLS or XLSX spreadsheet to finish.
 
-NOTE: Only the first worksheet in a spreadsheet workbook will be processed as a decision table when you upload the spreadsheet in {CENTRAL}. Each `RuleSet` name combined with the `RuleTable` name must be unique across all decision table files in the same package.
+[NOTE]
+====
+
+Only the first worksheet in a spreadsheet workbook is processed as a decision table when you upload the spreadsheet in {CENTRAL}. Each `RuleSet` name combined with the `RuleTable` name must be unique across all decision table files in the same package.
+
+If you want to process multiple worksheet decision tables, then create a `.properties` file with the same name as the spreadsheet workbook. The `.properties` file must contain a property sheet including the comma-separated names of the worksheets as follows:
```

Comment:
Suggestion - apply "To do this, do that" and clarify the mechanism:

s/If you want to process multiple worksheet decision tables, then create a `.properties` file with the same name as the spreadsheet workbook./To process multiple worksheets as decision tables, create a `.properties` file with the same name as the spreadsheet workbook.

Suggestion - does the `.properties` file uses comma-separated values, if so, adopt the naming as below. Change to using "for example:":

s/The `.properties` file must contain a property sheet including the comma-separated names of the worksheets as follows:/The `.properties` file must contain a property sheet with comma-separated values (CSV) for the names of the worksheets, for example:

Reply:
Done